apps/wfo-ui/package.json

```diff
@@ -8,7 +8,8 @@
         "tsc": "tsc --noEmit",
         "build": "next build",
         "start": "next start",
-        "lint": "next lint"
+        "lint": "next lint",
+        "reset": "rm -rf node_modules"
```

Comment:
This does not seem to be needed for normal usage of the project. If still yes, maybe rename to "uninstall"